### ISSUE TITLE
Local config messes with global config

### ISSUE THREAD

**Issue by cigix** [PyCQA/pylint#1265]
It seems local message control configuration overrides global configuration before it is even set.
### Steps to reproduce
test.py:
```python
import invalid1
# pylint: disable=import-error
import invalid2
```
pylintrc:
```
[MESSAGE CONTROL]                                                                   
disable=I,missing-docstring,unused-import,import-error                              
[REPORTS]                                                                        
reports=no
```
Also happens when 'import-error' is disabled from the command line instead of pylintrc.
### Current behavior
```
************* Module test
E:  1, 0: Unable to import 'invalid1' (import-error)
```
### Expected behavior
Nothing since import errors are disabled globally.

### pylint --version output
pylint 1.6.4, 
astroid 1.4.9
Python 3.5.1 (default, Dec 09 2015, 07:29:36) [GCC]
